browser/resources/settings/brave_overrides/system_page.ts

```diff
@@ -36,8 +36,8 @@ RegisterPolymerTemplateModifications({
       // default route, rather than always showing, otherwise when we navigate,
       // we'll get all the toggles showing up.
       const nonStyleChildren = (
-        Array.from(templateContent.children) as HTMLElement[]
-      ).filter((t) => t.tagName !== 'STYLE')
+        Array.from(templateContent.children)
+      ).filter((t: any) => t.tagName !== 'STYLE')
```

Comment:
Can you remove the `any` here? If `as HTMLElement[]` was triggering the `no-unecessary-type-assertions` I suspect it actually has the type :smile: